client/src/components/userMenu/userMenu.jsx

```diff
@@ -129,12 +128,11 @@ export class UserMenu extends React.Component {
 					</div>
 					{this.state.showSideBar === true ? this.createSideBar() : null}
 				</div>
-				<div id="rightSideSpacer"></div>
-				<div className="floatingButtons">
-					<div id="floatingContainer">
+{/*				<div className="floatingButtons">
+					<div className="floatingContainer">
 						{this.state.showFloatingButtons === true ? this.showFloatingButtons() : null}
 					</div>
-				</div>
```

Comment:
This probably isn't needed anymore, right? Floating buttons are now part of index.jsx